RELEASE_NOTES.md

```diff
@@ -95,10 +95,24 @@ CSM 1.2 contains approximately 2000 changes spanning bug fixes, new feature deve
 
 ### Base platform component upgrades
 
-* `istio` V1.8 from V1.7
-* Loftsman V1.2.0-1
-* Kubernetes V1.20.13 from V1.19.9
-* Ceph V15.2.15 from V15.2.8
+* `istio V1.8`
```

Comment:
This could be more readable as a table:

  | Platform Component           | Version      |
  |------------------------------|--------------|
  | Istio                        | 1.8          |
  | Loftsman                     | 1.2.0-1      |
  | Kubernetes                   | 1.20.13      |
  | Ceph                         | 15.2.15      |
  | Containerd                   | 1.5.7        |
  | CoreDNS                      | 1.7.0        |
  | Etcd for Kubernetes          | 3.5.0        |
  | Etcd cluster                 | 3.3.22       |
  | Helm                         | 3.2.4        |
  | Kiali                        | 1.28.1       |
  | Keepalived                   | 2.0.19       |
  | MetalLB                      | 0.11.0       |
  | Multus                       | 3.7          |
  | PostgreSQL                   | 12.11        |
  | Strimzi Operator             | 0.27.1       |
  | Vault                        | 1.5.5        |
  | Vault Operator               | 1.8.0        |
  | Zookeeper                    | 3.5.9        |